src/blocks/modal.css

```diff
@@ -185,6 +210,11 @@
   .modal__container-form-description {
     width: 238px;
   }
+
+  .modal__container-form-input {
+    width: 238px;
+  }
+
   .modal__container-form-button {
```

Comment:
Needs correction: The cursor of the submit button should be `pointer` if the button is not disabled

Reply:
Okay I added the code for this